common/artifacts/000_wc_artifacts.txt

```diff
@@ -11,6 +11,10 @@ slots = {
 	ceremonial_torso = 1
 	# mask = 1
 
+	shoulder = 1
+	waist = 1
+	legs=1
+	feet=1
```

Comment:
The additional slots add granularity to the item system. Less significant slots can provide small cumulative bonuses, useful for characters not using plate armor. Magic-based characters in particular would benefit given the prevalence of enchanted items in the setting, i.e. utility/special effect items. Clown characters shouldn’t be a visible problem since none of the apparel is shown on the 3D model (afaik), and hero units in WC3/FT made use of minor loot, so it’s not strictly a MMO thing either. Lore-wise, both bookmarks have cataclysmic events that wipe out entire armies and even kingdoms, so mismatching equipment wouldn't be too surprising, especially since the magical properties of a given item are more important than the metal they're made of.
Given that this is the most popular CK2 mod atm there’ll probably be plenty of players from the MMO too, and using an equally detailed inventory system provides them the ability to roleplay as one of the established characters. Even if some of the items aren’t particularly useful gameplay-wise, they can still serve an immersive/AAR purpose, so people into that sort of thing will still enjoy them (i.e. location-specific WoW quest items with lore descriptions and tiny bonuses). If I put any of those in I’ll keep them destroyable though, that way they won’t clutter the inventory of players not interested in them.

Reply:
>The additional slots add granularity to the item system.

Granularity is fine, but too much granularity causes only pain, and if granularity causes problems, system needs to be simplified. Most of the new slots overlaps with an existing torso slot, anyway.

>Less significant slots can provide small cumulative bonuses, useful for characters not using plate armor.

Additional slots don't give anything to character that not use plate armor. Cloth/Leather torso? Yes, it will give something for characters, that don't use plate armor. Additional slots? No.

>Magic-based characters in particular would benefit given the prevalence of enchanted items in the setting, i.e. utility/special effect items.

Items with special effects can be added without adding additional slots. **Nothing** will happens if you replace, for example, magical bear-summoning belt with bear-summoning spellbook.

>detailed inventory system provides them the ability to roleplay as one of the established characters

I don't think players than can't roleplay without boots from MMO can roleplay at all.

>Clown characters shouldn’t be a visible problem since none of the apparel is shown on the 3D model (afaik), and hero units in WC3/FT made use of minor loot, so it’s not strictly a MMO thing either. 

For example, AI is stupid enough to wear cloth robe + plate boots+ plate shoulder, which, if character is not insane, will be immersion breaking. You can apply class restrictions for plate/cloth items, but characters with both magical and physical classes, like Daelin or Anasterian still will have problems if they will have both plate and cloth pieces in their inventory.

>so mismatching equipment wouldn't be too surprising

Can you imagine craftsman who craft for his employer only shoulders, but no armor? If smith craft you body armor, why he craft for you only torso and not torso+shoulders+leg armor+belt, etc? If you want to change this, you need to spend tremendous amount of time for not really important mechanic.

---

Anyway, I think that if you wish to add additional slots, we need to have a serious talks about artifacts design in the mod with dev team first and gather others :thonk:-s, since, from design perspective, it's pretty important decision - do we really need minor artifact slots or it will be just third wheel? Do we want to invest our time and energy in it? And if answer will be "yes", it will require a separate branch.

---

Aside from complaints, nice work, tho

Reply:
I agree with @MemeWizard. It's just not important for the lore. Plus, I wouldn't add new slot if there's only one artifact.

Reply:
> For example, AI is stupid enough to wear cloth robe + plate boots+ plate shoulder, which, if character is not insane, will be immersion breaking. You can apply class restrictions for plate/cloth items, but characters with both magical and physical classes, like Daelin or Anasterian still will have problems if they will have both plate and cloth pieces in their inventory.

I was hoping that flags could be used so that certain classes would stick to particular types, even if they could technically could equip better ones (presumably there is a lore reason why spell classes avoid thicker armour). I see your point though, dual class problems will be a mess, one I hadn't thought of, and if the characters don't have anything else to equip there it becomes jarring either way.

> Can you imagine craftsman who craft for his employer only shoulders, but no armor? If smith craft you body armor, why he craft for you only torso and not torso+shoulders+leg armor+belt, etc? If you want to change this, you need to spend tremendous amount of time for not really important mechanic.

A blacksmith decision (if vanilla not already used) was going to allow sets of armour to be crafted (i.e. Steel, Thorium), and tailor/leather equivalents could cover other classes.


> I agree with @MemeWizard. It's just not important for the lore. Plus, I wouldn't add new slot if there's only one artifact.

I was going to gradually add most of the items the characters were shown having in WoW, so all of the added slots would have had plenty of items. I will agree that some of the slots wouldn't add much; belts could easily use ceremonial torso, and shoulders were contentious but I figured that some races/classes would use them for a minor prestige gain (i.e. Orc commanders).

Given the issues with the new slots, they might be better suited to a submod, with aspects being mainlined if they make sense for everyone. I'll keep any mainline additions in the existing slots. The last thing I want to do is slow down work on other dev projects with design concerns.